types/bootstrap/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for bootstrap 5.0
+// Type definitions for bootstrap 5.0.2
```

Comment:
```suggestion
// Type definitions for bootstrap 5.0
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library